google/ggrc-core
# GGRC-3710 Cycles, Cycle Task Groups and Cycle tasks should be filtered by 'In Progress'

# Issue description

`InProgress` state should be replaced with `In Progress`.

# Steps to test the changes

AC1. Cycles, Cycle Task Groups, Cycle Tasks should have state "In Progress' instead of "InProgress'.
All these objects should be filtered / searched / imported / exported by "In Progress" state.
Filtering should work on:
* MyTasks page
* Global Search

AC2. On UI Cycles, Cycle Task Groups, Cycle Tasks should have state "In Progress":
- in tree view;
- on bulk update pop-up;
- in warnings;
- advanced search.

# Solution description

Changed `InProgress` state with `In Progress`.

# Sanity checklist




# PR Review checklist
